pydevd_file_utils.py

```diff
@@ -231,8 +232,28 @@ def _NormPaths(filename):  #Let's redefine _NormPaths to work with paths that ma
     #Don't fail if there's something not correct here -- but at least print it to the user so that we can correct that
     traceback.print_exc()
 
-
-if PATHS_FROM_ECLIPSE_TO_PYTHON:
+norm_file_to_client = None
```

Comment:
I think they should start with the default value here, just to make it clearer what's the proper value (even if it's reassigned later on in setup_client_server_paths).